debian/package.json

```diff
@@ -3,12 +3,14 @@
   "version": "1.6.0",
   "description": "Discover and collect music on YouTube",
   "main": "index.js",
+  "homepage": "http://headsetapp.co",
   "scripts": {
     "start": "NODE_ENV=development NODE_PATH=. nf start",
     "electron": "./node_modules/.bin/electron .",
     "player": "./bin/player-window",
     "production": "NODE_ENV=production electron .",
-    "build": "NODE_ENV=production NEW_VERSION=$npm_package_version ./bin/build"
+    "build": "NODE_ENV=production NEW_VERSION=$npm_package_version ./bin/build",
+    "rebuild": "electron-rebuild"
```

Comment:
It's just a shortcut for rebuilding `dbus` module. This has to happen only once after running `npm install`. Feel free to delete the line if you want to. I won't make a difference